corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowserMain.java

```diff
@@ -0,0 +1,58 @@
+package org.corfudb.browser;
+
+import java.util.Map;
+
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.util.GitRepositoryState;
+import org.docopt.Docopt;
+
+/**
+ * Main class for the UFO Browser Tool.
+ * Command line options are documented in the USAGE variable.
+ *
+ * - Created by pmajmudar on 10/16/2019
+ */
+public class CorfuBrowserMain {
+    private static final String USAGE = "Usage: corfu-browser --host=<host> " +
```

Comment:
Btw, if you wan't an alternative to docopt, you can consider jcommander. I feel like its more cleaner way to specify command args.

Reply:
We decided to use docopt because it is being used elsewhere in the repo.